src/signals/settings/users/containers/Detail/components/UserForm/__tests__/UserForm.test.js

```diff
@@ -27,14 +42,16 @@ describe('signals/settings/users/containers/Detail/components/UserForm', () => {
 
   it('should make fields disabled', () => {
     const { container, rerender, queryByText } = render(withAppContext(<UserForm />));
-    const numFields = container.querySelectorAll('input').length;
+    // const numFields = container.querySelectorAll('input').length;
```

Comment:
forgot to remove?

Reply:
No, this still needs to be there for an upcoming PR. See comment a couple of lines down.